src/commands/config.ts

```
@@ -270,10 +329,101 @@ function parseJsonObject(opts: {
 }
 
 function parseKeyPath(opts: { readonly raw: string }): readonly string[] {
-  return opts.raw
-    .split(".")
-    .map(part => part.trim())
-    .filter(part => part.length > 0)
+  const parts: string[] = []
+  let buffer = ""
+  let escape = false
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)